### PR TITLE
BETA: Register mayank.is-a.dev

### PR DESCRIPTION
**Pull request by MayankServers**
Added `mayank.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by MayankServers** [comment]
@wdhdev

File: domains/mayank.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "MayankServers",
+        "email": "mayankreal657@gmail.com"
+    },
+    "record": {
+        "CNAME": "mayankserver.github.io"
```
Comment (CuteDog5695) [inline domains/mayank.json:7]: site gives 404 page

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/mayank.json (hunk 1 above)
Comment (MayankServers) [reply domains/mayank.json]: > site gives 404 page

Sorry wrong url

Comment (MayankServers) [reply domains/mayank.json]: @CuteDog5695 recheck it

**Review by iostpa — APPROVED** [review]
seems good to me now

**Pull request merged**